runtimes/huggingface/README.md

```diff
@@ -55,6 +55,36 @@ MLSERVER_MODEL_HUGGINGFACE_OPTIMUM_MODEL=true
 ```
 ````
 
+### Loading local models
+It is possible to load a model from a local path into a HuggingFace pipeline, if desired. 
+The path must point to a folder containing the model artefacts compatible with the HuggingFace specifications.
+#### Multi-model serving folder structure
+In the case of MMS, the example below assumes the desired model folder resides in `./models`, e.g. `./models/your-model-artefacts-folder` and inside a `models-settings.json` file is expected to exist.
+The `uri` field could be an absolute path to the model artefacts folder or a relative path to the folder from which MLServer is started.
+```{code-block} json
+---
+emphasize-lines: 5-5
+---
+{
+  "name": "qa",
+  "implementation": "mlserver_huggingface.HuggingFaceRuntime",
+  "parameters": {
+      "uri": "./models/your-model-artefacts-folder",
+      "version": "v1.0",
+      "extra": {
+          "task": "question-answering",
+      }
+  }
+}
+```
```

Comment:
I think this should be already covered on other sections of the docs (i.e. both around MMS and also around relative paths, etc.) so I'd suggest removing it in favour of the other sections.